adding virool to the druid-powered page

rmustafin opened the pull request:
pretty simple, eh? :)

BTW got an error with jekyll (it couldn't load redcarpet) event with a new gems set and could solve it only adding redcarpet to the Gemfile, shouldn't we?

fjy commented:
@rmustafin Awesome!

BTW are you guys really doing 1B events/min? It that per hour or day by any chance?

rmustafin commented:
@fjy I'm so sorry! It's a number of events per day!!! I wanted to specify number of events per minute but thought 1B number is more impressive and changed the number but not "per minute" :( We moved only part of our analytics to druid and send only 10% of RTB events so it's promising to be much more events. Sorry about that, will make another PR cause that one is closed.
Happy 4th of July!
